docs/docs-content/integrations/k3s.md

```diff
@@ -0,0 +1,143 @@
+---
+sidebar_label: 'K3s'
+title: 'K3s'
+description: 'K3s pack in Palette Edge'
+hide_table_of_contents: true
+type: "integration"
+category: ['kubernetes', 'amd64', 'fips']
+sidebar_class_name: "hide-from-sidebar"
+logoUrl: 'https://registry.spectrocloud.com/v1/edge-k3s/blobs/sha256:5b7f8a80330d6938056848905238efbbd085e2f0f86fd03d7be10ccdabf791c7?type=image/png'
+tags: ["packs", "kubernetes", "k3s", "edge"]
+---
+
+K3s is a lightweight distribution of Kubernetes, specifically designed for resource-constrained environments and IoT applications. Unlike standard Kubernetes with multiple binaries for its various components, K3s is packaged as a single binary with a lightweight storage backend and minimal external dependencies. For more information about K3s, refer to [K3s documentation](https://docs.k3s.io/). 
+
+## Versions Supported
+
+<Tabs queryString="versions">
+<TabItem label="1.27.X" value="k3s_1.27">
+
+### Prerequisites
+
+- An edge device with AMD64(x86_64) processor architecture or a Palette Virtual Cluster. 
+- A minimum of 2 CPU cores and 1 GB memory. 
+
+### Parameters
+
+Since you can deploy both Palette Virtual Clusters and Edge clusters using K3s, you have different configuration options depending on the type of cluster.
```

Comment:
```suggestion
Since you can deploy both virtual clusters and Edge clusters using K3s, you have different configuration options depending on the cluster type.
```

Reply:
The first occurrence within a doc:  Palette Virtual Cluster

Subsequent occurrences within the same doc: virtual cluster